src/common/security/db/context.go

```diff
@@ -13,3 +13,76 @@
 // limitations under the License.
 
 package db
+
+import (
+	"github.com/vmware/harbor/src/common/models"
+	"github.com/vmware/harbor/src/ui/pms"
+)
+
+// SecurityContext implements security.Context interface based on database
+type SecurityContext struct {
+	user *models.User
+	pms  pms.PMS
```

Comment:
It maybe OK us pms as a pointer?  As in DB mode, pms can be a system wide variable?